docs/guides/importing.md

```diff
@@ -0,0 +1,53 @@
+---
+page_title: "Importing existing Power Platform resources"
+subcategory: "Guides"
+description: |-
+  <no value>
+---
+
+# Importing existing Power Platform resources
+
+Many of the resources in the Power Platform provider can be imported into your Terraform configuration.  This guide will show you how to import existing resources into your configuration.  Resources that can be imported have an `import` section in their documentation.
+
+## Using `terraform import`
+
+Using `terraform import` will only import resources into the state.  It does not generate configuration and requires that you write the configuration yourself and ensure that it is correct.
+
+```bash
+terraform import powerplatform_environment.example_env 00000000-0000-0000-0000-000000000001
+```
+
+## Using import blocks
+
+Terraform allows the use of import blocks to import existing resources into your configuration.  This can be useful if you have pre-existing resources that you want to manage with Terraform.  The following is an example of an import block used to import the default Power Platform environment:
+
+```terraform
```

Comment:
This sample don't work for me I'm trying to import a environment but doesn't work.

Error: Configuration for import target does not exist
│
│   on main.tf line 17, in import:
│   17:   to = powerplatform_environment.example_env
│
│ The configuration for the given import target powerplatform_environment.example_env does not exist. If you wish to automatically generate config for this resource, use the
│ -generate-config-out option within terraform plan. Otherwise, make sure the target resource exists within your configuration. For example:
│
│   terraform plan -generate-config-out=generated.tf

Reply:
OK the configuration of the environment that I want to import should match with the resource.